ios/RNPdftron.m

```diff
@@ -142,6 +167,36 @@ - (NSError *)errorFromException:(NSException *)exception
             }];
 }
 
++(NSString*)exportAsImageHelper:(PTPDFDoc*)doc pageNumber:(int)pageNumber dpi:(int)dpi exportFormat:(NSString*)exportFormat
+{
+    NSString * resultImagePath;
+    BOOL shouldUnlock = NO;
+    @try {
+        [doc LockRead];
+        shouldUnlock = YES;
+
+        PTPDFDraw *draw = [[PTPDFDraw alloc] initWithDpi:dpi];
+        NSString* tempDir = NSTemporaryDirectory();
+        NSString* fileName = [NSUUID UUID].UUIDString;
+        resultImagePath = [tempDir stringByAppendingPathComponent:fileName];
+        resultImagePath = [resultImagePath stringByAppendingPathExtension:exportFormat];
+        PTObjSet* hintSet = [[PTObjSet alloc] init];
+        PTObj* encoderHints = [hintSet CreateArray];
+        [encoderHints PushBackName:@"JPEG"];
```

Comment:
This code seems to be ignoring the requested `exportFormat` and always exporting as JPEG.